AjobK/Athena
# 388 verify account v1

closes #388 

<br />

 **What is it supposed to do**  
Implements the account verification flow, generating a verification link after registering. It creates a verification entity for an account when it is registered, and deletes it when the generated link is visited. The link is functional, but is not sent to the user's email address. A temporary button with the link is added so you can actually verify.

<br />

 **How can we test your branch**  
DB was updated so do npm run reset first. Register an account, after which "verification email sent" should be shown. Click on the temporary verify button, which should verify your account and sign you in and redirect you to your new profile. If you go to the link after the account has been verified already (or the code is incorrect), a message should appear stating that the code is invalid.

<br />

**1. General checklist**

<br />

**2. If relevant, front-end checklist**

<br />

**3. If relevant, back-end checklist**

<br />

**4. If relevant, test these browsers**
